### DISCUSSION TITLE
How to use edge-tts with goldendict?

### DISCUSSION THREAD

**Discussion by Giancarlo-Ma** [rany2/edge-tts#94] [Q&A] [answered]
Anyone has tried this?

**Comment by rany2** [comment]
Made this issue: https://github.com/goldendict/goldendict/issues/1648

**Comment by shenlebantongying** [comment, marked as answer]
The related functionality already exists:

Just add `edge-playback --text "%GDWORD%"` as a custom program and set the type to `audio`.

![image](https://github.com/rany2/edge-tts/assets/20123683/3cb964ba-7376-4a1b-9f24-89470796d702)

**Reply by rany2** [reply]
@shenlebantongying I had to move it to Q/A first, but thank you! I tried before making this issue and the setting wasn't really obvious; though admittedly it might be obvious to someone who has used goldendict for a bit more than a few mins :)

**Reply by shenlebantongying** [reply]
Thank you for making edge-tts :D

**Reply by Shikho18** [reply]
> Thank you for making edge-tts :D

(｡♥‿♥｡)shikho18$
